apps/api/src/domain/models/context/edge-connection.entity.ts

```diff
@@ -95,6 +86,26 @@ export class EdgeConnectionMember<T extends EdgeConnectionContext = EdgeConnecti
 export class EdgeConnection extends Aggregate {
     type = AggregateType.note;
 
+    @ExternalEnum(
+        {
+            enumName: 'EdgeConnectionType',
+            enumLabel: 'Edge Connection Type',
+            labelsAndValues: [
+                {
+                    label: 'note',
+                    value: EdgeConnectionType.self,
+                },
+                {
+                    label: 'connection',
```

Comment:
I see, this is the differentiation here.  I guess the potentially confusing thing is that, if I understand correctly, both are types of connections, and both entail creating a note.  It would be good to discuss this further.

Reply:
Yea. We really need to resolve this. Let's have a brainstorming meeting to discuss further.